src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -213,6 +213,24 @@ if (!isLoggedIn(cookie)) {
 </html>
 ```
 
+### `Astro.locals`
+
+The `Astro.locals` object is a user-defined object that can be used to access data from your Astro middleware or your server inside your page. This can be useful for passing per-request data like user sessions.
```

Comment:
```suggestion
`Astro.locals` is an object that can be used to access data from your [Astro middleware](/en/guides/middleware/) or your server inside your page. This can be useful for passing per-request data like user sessions.
```